<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>401</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>401</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>659</width>
    <height>401</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0.5 rgba(0, 0, 0, 255), stop:0.515789 rgba(198, 176, 135, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(188, 167, 128, 0);
font: 81 20pt &quot;Rockwell Extra Bold&quot;;

</string>
   </property>
   <property name="text">
    <string>LOGIN DO SISTEMA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Arial&quot;;
background-color: rgb(198, 176, 135, .0);
image: url(:/imgs/icons/user.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(198, 176, 135, 0);
image: url(:/imgs/icons/lock.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>225</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 10;
	background-color: rgb(0,0,0);
	border: 5 solid black;
	color: rgb(192, 171, 131);
	font: 81 7pt &quot;Rockwell Extra Bold&quot;;
}

QPushButton:pressed {
	background-color: rgb(0, 0, 0, 0.7);
	border: 5 solid rgb(0, 0, 0, 0);

}
</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>145</y>
     <width>199</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 8pt &quot;Noto Serif&quot;;
border-radius: 10;
padding-left: 30;</string>
   </property>
   <property name="placeholderText">
    <string>| Digite seu login</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>185</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
padding-left: 30;
border-radius: 10;
font: 75 8pt &quot;Noto Serif&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>| Digite sua senha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>181</y>
     <width>80</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 176, 135,0);
font: 24pt &quot;Algerian&quot;;

color: rgb(198, 176, 135);</string>
   </property>
   <property name="text">
    <string>Duda</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 176, 135,0);
image: url(:/imgs/icons/manequim.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>211</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 176, 135,0);
color: rgb(230, 230, 230);
font: 11pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string>Fashion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>173</y>
     <width>55</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 176, 135,0);
image: url(:/imgs/icons/guarda-roupa.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
	background-color: rgb(255, 170, 0, 0);
	image: url(:/imgs/icons/fechar.png);
}

QPushButton:hover {
	image: url(:/imgs/icons/fechar_red.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>0</y>
     <width>25</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
	background-color: rgb(255, 170, 0, 0);
	image: url(:/imgs/icons/minimize-o-sinal.png);
}

QPushButton:hover {
	image: url(:/imgs/icons/minimize_red.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_8</zorder>
  <zorder>label_11</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
